pkg/scheduler/apis/config/v1alpha2/defaults.go

```diff
@@ -156,3 +156,9 @@ func SetDefaults_KubeSchedulerConfiguration(obj *v1alpha2.KubeSchedulerConfigura
 		obj.EnableContentionProfiling = &enableContentionProfiling
 	}
 }
+
+func SetDefaults_InterPodAffinityArgs(obj *v1alpha2.InterPodAffinityArgs) {
+	if obj.HardPodAffinityWeight == nil {
+		obj.HardPodAffinityWeight = pointer.Int32Ptr(1)
```

Comment:
perhaps worth defining the default here: https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/apis/config/types.go#L28 since you are using "1" as a default in multiple places, but feel free to ignore.

Reply:
sg